plex_trakt_sync/main.py

```diff
@@ -189,16 +189,21 @@ def process_show_section(s, watched_set, listutil):
     logging.info("Now working on show section {} containing {} elements".format(s.title, len(allShows)))
     for show in allShows:
         guid = show.guid
+        if guid.startswith('plex://show/'):
+            if len(show.guids) > 0:
+                logging.debug("trying first alternative guid: " + str(show.guids[0].id))
+                guid = show.guids[0].id
+        x = provider = None
         if guid.startswith('local') or 'agents.none' in guid:
             # ignore this guid, it's not matched
             logging.warning("Show [{} ({})]: GUID is local, ignoring".format(
                 show.title, show.year))
             continue
-        elif 'thetvdb' in guid:
```

Comment:
why have you removed `thetvdb`? please add new scenarios, do not remove old scenaros.

Reply:
Sorry, this is literally my first time doing a pull request so I'm just fumbling through this. But my understanding was that the new TV agent no longer uses `thetvdb` which is why I removed it.

Reply:
@BEisem but there's no intention to stop supporting the previous tv agents. legacy and previous agents can be removed when they can't be used, but even the latest PMS includes the legacy agents.

Reply:
Plex metadata for the new agents can contain multiple guids, seems like imdb is the first and therefore used first in the elif-statement.

Example from my library:
`grandparentGuid="plex://show/5d9c080602391c001f57f2ce"` with guids:
```xml
<Guid id="imdb://tt5312444"/>
<Guid id="tmdb://1218828"/>
<Guid id="tvdb://5632569"/>
```
I have not seen metadata without an IMDB refenrence, but might still happen, so keeping the other loops is a good idea.

Reply:
Sorry for my ignorance, but where do you go to find this information? I would like to try and take a look at one of the episodes in my library that is causing this error, to see if I see the multiple guids that you have found, or if it is missing the IMDB reference.

Reply:
Scratch that last question, I found it in the XML of the episodes.

So I discovered that, for any episode that is throwing this error in PlextraktSync, the metadata for that episode does not contain any references to IMDB, TMDB or TVDB. It's as if the metadata is incomplete. And instead of an entry like `guid="plex://episode/5de7db195e4b67001edc92de"` it will show something like `guid="local://4036"`.

Just so we are comparing apples and apples, do you have the series "American Dad?" If you take a look at Season 15 Episode 22, this is one of the ones that is displaying this problem. In the metadata, the guid is `guid="local://4036"`. This must be what is causing the problem.

Reply:
I checked TMDB and there is no ep.22, therefore the plex agent might have a problem. Could be differences in numbering (tv order & dvd order and the like).

Reply:
pardon me, but what error are you both talking about?

Reply:
Oops, sorry... I thought I was here: https://github.com/Taxel/PlexTraktSync/discussions/208